test/integration/azureSpecialProperties.spec.ts

```diff
@@ -12,6 +13,35 @@ import {
   HttpHeaders
 } from "@azure/core-http";
 
+describe.only("auth validation", () => {
+  it("should add authorization header", async () => {
```

Comment:
This test case looks fine. But, is it really related to the lroPolicy changes?

Reply:
One of the side effects of having a faulty requestPolicyFactories option was that the default authentication policy was being inadvertently bypassed. This test is here to make sure we don't regress again in building the expected default auth pipeline